packages/twenty-server/src/engine/metadata-modules/field-metadata/dtos/field-metadata.dto.ts

```diff
@@ -147,7 +147,7 @@ export class FieldMetadataDTO<
 
   @IsBoolean()
   @IsOptional()
-  @Field()
+  @Field({ nullable: true })
```

Comment:
this is nullable in the input but non-null in db, as it is for objectMetadata. I really need to double check but I think we want to allow null values in the input that we backfill in the service